annotation-utils/src/main/java/eu/europeana/annotation/config/AnnotationConfiguration.java

```diff
@@ -67,6 +69,8 @@
     public static final String VALIDATION_ADMIN_SECRET_KEY = "adminsecretkey";
 
     public static final String API_KEY_CACHING_TIME = "annotation.apikey.caching.time";
+    
+    public static final String ETAG_FORMAT = "application/json";
```

Comment:
please remove this constant and use the existing constant WebAnnotationFields.FORMAT_JSONLD

Reply:
constant replaced